### ISSUE TITLE
Notifications for new Releases

### ISSUE THREAD

**Issue by srueg** [helm/monocular#608]
I would be interested in receiving notifications for new Helm chart releases. This could be email and (Slack) webhook based. Is this something that fits into the road map of monocular and maybe even the Hub?

**Comment by prydonius** [comment]
Thanks for the suggestion, I think this could be an interesting fit in the Hub as a sort of "watch" or "subscribe" feature on a chart.

**Comment by srueg** [comment]
Yes, exactly. I'm having a hard time to keep up with chart updates, or even noticing them.
